dockerfiles/Dockerfile.x86_64-debug

```diff
@@ -0,0 +1,96 @@
+FROM busybox:stable as busybox-include
+FROM amd64/debian:bullseye-slim as builder
+
+# Fluent Bit version
+ENV FLB_MAJOR 1
+ENV FLB_MINOR 9
+ENV FLB_PATCH 0
+ENV FLB_VERSION 1.9.0
+
+ARG FLB_TARBALL=https://github.com/fluent/fluent-bit/archive/v$FLB_VERSION.tar.gz
+ENV FLB_SOURCE $FLB_TARBALL
+RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log /tmp/fluent-bit/
+
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /tmp/fluent-bit/
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    curl \
+    ca-certificates \
+    cmake \
+    make \
+    tar \
+    libssl-dev \
+    libsasl2-dev \
+    pkg-config \
+    libsystemd-dev \
+    zlib1g-dev \
+    libpq-dev \
+    postgresql-server-dev-all \
+    flex \
+    bison \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -L -o "/tmp/fluent-bit.tar.gz" ${FLB_SOURCE} \
+    && tar zxfv /tmp/fluent-bit.tar.gz -C /tmp/fluent-bit --strip-components=1 \
+    && rm -rf /tmp/fluent-bit/build/*
+
+WORKDIR /tmp/fluent-bit/build/
+RUN cmake -DFLB_RELEASE=On \
+          -DFLB_TRACE=Off \
+          -DFLB_JEMALLOC=On \
+          -DFLB_TLS=On \
+          -DFLB_SHARED_LIB=Off \
+          -DFLB_EXAMPLES=Off \
+          -DFLB_HTTP_SERVER=On \
+          -DFLB_IN_SYSTEMD=On \
+          -DFLB_OUT_KAFKA=On \
+          -DFLB_OUT_PGSQL=On ..
+
+RUN make -j "$(getconf _NPROCESSORS_ONLN)"
+RUN install bin/fluent-bit /fluent-bit/bin/
+
+# Configuration files
+COPY conf/fluent-bit.conf \
+     conf/parsers.conf \
+     conf/parsers_ambassador.conf \
+     conf/parsers_java.conf \
+     conf/parsers_extra.conf \
+     conf/parsers_openstack.conf \
+     conf/parsers_cinder.conf \
+     conf/plugins.conf \
+     /fluent-bit/etc/
+
+# hadolint ignore=DL3006
+FROM debian:bullseye-slim
+LABEL description="Fluent Bit debug docker image" vendor="Fluent Organization" version="1.9.0" author="Eduardo Silva <eduardo@calyptia.com>"
+
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      libssl1.1 \
+      libsasl2-2 \
+      pkg-config \
+      libpq5 \
+      libsystemd0 \
+      zlib1g \
+      ca-certificates \
+      libatomic1 \
+      libgcrypt20 \
+      bash gdb valgrind build-essential \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /fluent-bit /fluent-bit
+
+COPY --from=busybox-include --chmod=555 /bin/busybox /bin/busybox
```

Comment:
@niedbalski not sure this is required - the current debug images seem to install busybox into a debian base image that also has a full shell and package manager. Any idea why?

Reply:
Personally I think we should remove the busybox usage across the board.